projects/mintswap/index.js

```diff
@@ -0,0 +1,28 @@
+const { addFundsInMasterChef } = require("../helper/masterchef");
+const { transformAvaxAddress } = require("../helper/portedTokens");
+
+const masterChefContract = "0xAdD22604caf79139450e9fb4851394fFCE1692Be";
+
+const avaxTvl = async (chainBlocks) => {
+  const balances = {};
+
+  const transformAddress = await transformAvaxAddress();
+  await addFundsInMasterChef(
```

Comment:
this seem to include MINT staked into the masterchef, would be better to separate that

Reply:
Do you mean, put the MINT address as ignore one in the addFundsInMasterChef method and then put it as staking part?